WordPress/Classes/Utility/App Configuration/ExtensionConfiguration.swift

```diff
@@ -0,0 +1,31 @@
+// WordPress Extension configuration
+
+import Foundation
+
+@objc extension AppConfiguration {
```

Comment:
As we will be having a couple of "similar-but-different" configuration files, (e.g., `WidgetConfiguration`, `AppConfiguration`, `AppConstants`), I think it would help to add some documentation/warning here (and in the counterpart file) stating that the **keys must be in-sync with the counterpart file**. 

While this doesn't entirely prevent mistakes, I feel it still helps to some extent to prevent potential crashes due to keys missing from the other file. More detail on this: https://github.com/wordpress-mobile/WordPress-iOS/pull/19479#discussion_r1002819063

Reply:
Thanks for linking this comment thread 👍.

Yes, we went for this solution without too many discussions, simply following the pattern that was already established with `AppConfiguration` files. However, it does have drawbacks with discoverability and the necessity to build both targets to ensure there're no errors.

Reply:
Good point @dvdchr!
I've added documentation here and in the other configuration files c9a58f22befd1902e4c78b19e8947def4e53f160 👍